rust/src/messages.rs

```diff
@@ -863,6 +885,22 @@ impl RouterCreateWebrtcTransportRequest {
             is_data_channel: true,
         }
     }
+
+    pub(crate) fn to_fbs(&self) -> web_rtc_transport::WebRtcTransportOptions {
+        web_rtc_transport::WebRtcTransportOptions {
+            base: Box::new(transport::Options {
+                direct: false,
+                max_message_size: 0,
```

Comment:
Why do we have these fields in "base" options that do not make sense to generic transport and set to fake values?

Reply:
I'm setting this to optional. Both 'direct'  and 'max_message_size'  are used by the base Transport class and only applied to direct transport.

I'm setting it to optional in this PR.

Reply:
AFAIR we inspect and use those fields in Transport.cpp so they need to be in the base options of message.